.github/workflows/ci.yml

```diff
@@ -90,20 +90,20 @@ jobs:
       - name: Install Terraform
         uses: hashicorp/setup-terraform@v3
         with:
-          terraform_version: 1.2.5
+          terraform_version: 1.11.4
```

Comment:
why do we need this?

Reply:
I upgraded the terraform version used for terragrunt tests.